src/test/java/org/reactome/release/ModelsTSVParserTester.java

```diff
@@ -81,7 +81,7 @@ public void testFileContainsCorrectMapping() {
         Map<String, Set<String>> pathwayToBioModelsIds = ModelsTSVParser.parse(filepath);
 
         // Tests pathway contains correct BioModels id
-        assertThat(pathwayToBioModelsIds.get("R-HSA-199418"), contains("BIOMD0000000175"));
+        assertThat(pathwayToBioModelsIds.get("R-HSA-199418"), hasItem("BIOMD0000000175"));
```

Comment:
I'm curious why this was done. What's wrong with `contains`?

Reply:
`contains` requires all elements in the collection be specified in order to match where `hasItem` matches for any of the items.

R-HSA_199418 has more than one BioModels identifier mapped to it, so `hasItem` is needed to match a single one.  Alternatively, we could provide all identifiers and keep `contains`.  That might be a good way to go, too.